src/test/java/uk/gov/hmcts/sptribs/systemupdate/event/SystemMigrateCaseFlagsTest.java

```diff
@@ -70,15 +74,73 @@ void shouldSuccessfullyUpdateCaseFlags() {
         systemMigrateCaseFlags.setMigrationFlagEnabled(true);
 
         //When
-        AboutToStartOrSubmitResponse<CaseData, State> response =
+        final AboutToStartOrSubmitResponse<CaseData, State> response =
+            systemMigrateCaseFlags.aboutToSubmit(updatedCaseDetails, beforeDetails);
+
+        //Then
+        assertNotNull(response.getData());
+        assertNotNull(response.getData().getCaseFlags());
+        assertNotNull(response.getData().getSubjectFlags());
+        assertNotNull(response.getData().getRepresentativeFlags());
+        assertNotNull(response.getData().getApplicantFlags());
```

Comment:
There's a couple problems with the shouldSuccessfullyUpdateCaseFlags test.
- beforeDetails and updatedCaseDetails have the same caseData so I can't tell if there have been any changes
- Checking if the flags are not null is not very useful. It would be more useful to test that the expected flags have been returned.